### ISSUE TITLE
It's a real-time app?

### ISSUE THREAD

**Issue by francisrod01** [ccoenraets/directory-ionic-nodejs#4]
Hy guys,

This is application is the real-time with ionic and nodejs and mongodb?

**Comment by HackMasterC** [comment]
No, the database is in the file: employees.js. I have the issue that is not real time :( .

**Comment by harshsanghani** [comment]
Hello

I have one issue, I have install the ionic application using the command and then copy the WWW directory from the repository and installed the npm dependency and its working perfectly in the browser with node server.

Now I have created the android application from this using add ionic plateform, now this created the default ionic application as the android APK, It does not create the working employee-directory application for the android.

can you please ask me, what is wrong in this? Why the android APK is not the eompyee-directory?
